### PR TITLE
Added advancetimer feature

### PR DESCRIPTION
**Pull request by gstarrett2**
Timer feature added.  Just a little cleaner than my original submission (consolidated some repetitive code).

### CODE REVIEW

**Comment by gstarrett2** [comment]
Forgot to mention, I fixed a bug that would show up if you had any spaces in your image names by quoting the URL values.  I've included that in this patch as well.

File: gallry.js [hunk 1]
```diff
@@ -254,16 +257,16 @@ var gallry = (function() {
               var str = arr[i].src;
               var first = str.slice(0, str.length - 4); // get first part of string
               var last = str.slice(str.length - 4, str.length); // get .jpg, .png, .svg, .gif files --- must be 4 character file extension
-              el[i].style.backgroundImage = "url(" + first + "_2x" + last + ")"; // defining gallery slide image via arr.src prop
+              el[i].style.backgroundImage = "url('" + first + "_2x" + last + "')"; // defining gallery slide image via arr.src prop
```
Comment (bwynn) [inline gallry.js:260]: Good catch on the single quotes! 👍

**Review by bwynn — APPROVED** [review]
Thank you for your contributions! I'll update on the various registries. 

If there are additional features you'd like to continue on, I'd be happy to add you as a contributor to this repository. That said, revisiting this library has me wanting to provide support to some more modern javascript, allowing for overall better library performance. Is that something that your team would/could benefit from? eg. `gallry.js` as a React or Vue component?

File: gallry.js (hunk 1 above)
Comment (gstarrett2) [reply gallry.js:260]: Thanks!  

I appreciate the offer, I'm not sure what I'd add to it (at least for now), but I'll certainly keep feeding back any improvements I make.  I was just looking for something simple / easy to implement, I don't know if a conversion to React or Vue would help (I'm only using the Bootstrap library currently for the website this is going on to, so it doesn't matter which library you're using).

Comment (gstarrett2) [reply gallry.js:260]: P.S.  Meant to say:  simple / easy to implement, and has the needed features!  There's a lot of simpler sliders out there that lack the nav features.

**Pull request merged**